ironfish/src/wallet/wallet.ts

```diff
@@ -701,10 +701,14 @@ export class Accounts {
             const noteMapValue = await sender.getDecryptedNote(unspentNote.hash)
             if (noteMapValue) {
               this.logger.debug(`Unspent note has index ${String(noteMapValue.index)}`)
-              await sender.updateDecryptedNote(unspentNote.hash, {
-                ...noteMapValue,
-                spent: true,
-              })
+              await sender.updateDecryptedNote(
+                unspentNote.hash,
+                {
+                  ...noteMapValue,
+                  spent: true,
+                },
+                null,
```

Comment:
This will break any notes that are being fixed up. It'll be marked as spent but it'll be set to an off chain note.

Reply:
You're right, we need to look up the transaction for the note here to get the right sequence.